recipes/rbdl-casadi/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "rbdl-casadi" %}
+{% set version = "2.6.0.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/pyomeca/rbdl-casadi/archive/{{ version }}.tar.gz
+  sha256: e07ab94e25ca99fdfa77341f6ea5bda5480bac3493e7af03158144da71c6a6eb
+
+build:
+  number: 0
+  skip: True  # [win or osx]
```

Comment:
You supply a build script for win but skip, intended?

Reply:
Yep, as soon as CasADi is avaible on conda-forge, I'll remove the skip